### ISSUE TITLE
Twitter - hide muted and blocked users

### ISSUE THREAD

**Issue by swyxio** [letsblockit/letsblockit#518]
### What website(s) would this filter apply to?

Twitter.com

### What content should be blocked?

https://twitter.com/levelsio/status/1683504409643302912

### What options should be offered?

![image](https://github.com/letsblockit/letsblockit/assets/6764957/e9f255b0-783c-4098-9c13-4f568757c323)

![image](https://github.com/letsblockit/letsblockit/assets/6764957/56dbd5b9-623d-46b7-8b18-6b2c80500773)

**Comment by JohnyP36** [comment]
As you can read at #466 we decided to remove twitter templates as they (almost) don't get used.

Also, see #269 and PR #302 that belongs to it: there is not really an maintainer or user that can help us keep that Twitter template up-to-date.

As I don't have an Twitter account I can't help you giving some rules you could use. In worst case you can give us a dump file with html code, so we can look up in the code what elements need to be blocked.

But lets wait on @xvello 's response.

**Comment by xvello** [comment]
Hello @swyxio, thanks for your interest in the project and for opening this issue.

As Jonny wrote, we indeed phased out our support for twitter last month due to dwindling usage and no response to our call for maintainer. I am not interested in revisiting that decision, so I'll close this issue as `wontfix`.

Don't hesitate to suggest other improvements though!
